### DISCUSSION TITLE
D3.js Visual

### DISCUSSION THREAD

**Discussion by kylewebster** [nbrown02/FlowViz#120] [Q&A] [answered]
Hi!

I'm facing an error on the Forecast page that "Uncertified visuals from AppSource, or added from a file, are not available due to your admin settings, unless the visual was added by the admin to the 'My Organization' page".

I spoke with our org admins who confirmed that we do not allow uncertified visuals.

Any idea how I might get this to work?

Thank you!

**Comment by nbrown02** [comment]
Hi there - unfortunately you’d have to rebuild this from scratch, or simply rely on using the desktop version to create forecasts as there is no quick way to do it. Sorry that’s all I can offer :(

**Reply by nbrown02** [reply, marked as answer]
@kylewebster @daneasy in the latest version I have changed the forecast visual to be a native visual bar chart rather than a d3 visual. This will fix the issue you were having :)

**Comment by daneasy** [comment]
hi, we also have the same issue, is there any recommended alternative visualisation we could use for this? Thanks!

**Reply by nbrown02** [reply]
Hi there - unfortunately you’d have to rebuild this from scratch, or simply rely on using the desktop version to create forecasts as there is no quick way to do it. Sorry that’s all I can offer :(
